### PR TITLE
Create sensor.pl.json

### PR DESCRIPTION
**Pull request by Antoni-Czaplicki**

### CODE REVIEW

File: custom_components/sleep_as_android/translations/sensor.pl.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+   "state": {
+     "sleep_as_android_status": {
```
Comment (IATkachenko) [inline custom_components/sleep_as_android/translations/sensor.pl.json:3]: double underscore needed.
`sleep_as_android__status`

File: custom_components/sleep_as_android/translations/sensor.pl.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+       "antysnoring": "Anty-chrapanie rozpoczęte",
+       "sound_event_snore": "Wykryto chrapanie",
+       "sound_event_talk": "Wykryto rozmowę",
+       "sound_event_cough": "Wykryto kaszel",
+       "sound_event_baby": "wykryto płacz dziecka",
+       "sound_event_laugh": "Wykryto śmiech",
+       "before_alarm": "Godzina przed alarmem"
+     }
+   }
+ }
```
Comment (IATkachenko) [inline custom_components/sleep_as_android/translations/sensor.pl.json:36]: Style: new line at EOF needed.

**Review by IATkachenko — CHANGES_REQUESTED** [review]

**Comment by IATkachenko** [comment]
Could you add empty line at the end of file? GitHub like it.

**Comment by Antoni-Czaplicki** [comment]
It is there
Screenshot_20220221-160702_Chrome

**Comment by IATkachenko** [comment]
Sorry, GitHub is a bit strange: new line exists, but not shown.
изображение

**Pull request merged**